kubehub/serializers/CloudProviderSerializer.py

```diff
@@ -1,17 +1,15 @@
 from rest_framework import serializers
-from . models import CloudProvider
+from ..models.CloudProvider import CloudProvider
```

Comment:
why double dots?

Reply:
for better code style we use directories for models, serializers and views instead of files thats why we need two dots for import